website/project/views/register.py

```diff
@@ -40,25 +38,18 @@
 from website.identifiers.client import EzidClient
 
 from .node import _view_project
-from .. import clean_template_name
-
 
 @must_be_valid_project
 @must_have_permission(ADMIN)
-@must_not_be_registration
 def node_register_page(auth, node, **kwargs):
 
-    ret = {
-        'options': [
-            {
-                'template_name': metaschema['name'],
-                'template_name_clean': clean_template_name(metaschema['name'])
-            }
-            for metaschema in OSF_META_SCHEMAS
-        ]
-    }
-    ret.update(_view_project(node, auth, primary=True))
-    return ret
+    if node.is_registration:
```

Comment:
@brianjgeiger and @JeffSpies are the recommended consults for all things route-related, and can best advise on an appropriate route name. 

I would note that underscores in URLs are relatively unusual; see discussion for example here: http://stackoverflow.com/questions/10302179/hyphen-underscore-or-camelcase-as-word-delimiter-in-uris

Also, note that I almost couldn't find this comment because it was attached to an outdated diff. You may want to create separate tickets to track the other concerns mentioned in the above. Let me know if other points need feedback.